.gitmodules

```diff
@@ -87,7 +87,8 @@
 	url = https://github.com/NVIDIA/cudnn-frontend.git
 [submodule "third_party/kineto"]
     path = third_party/kineto
-    url = https://github.com/pytorch/kineto
+    url = https://github.com/ROCm/kineto.git
+    branch = rocprof_sdk_rocm6.5_internal_testing
```

Comment:
I don't think we should have the branch name in the `.gitmodules` file. Instead we should bump the submodule commit as and when needed, to have a consistent commit be used for the submodule. So I'm removing this.